examples/Reading in linearised models.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from src.utils import plot_dfs
</pre><p>I think it looks nicer to keep all the imports at the top of the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/gEconpy/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>